docs/examples/IGMGalaxyField_examples.ipynb

```diff
@@ -0,0 +1,483 @@
+       "DEIMOS  PG1407_may_late 14:09:30.58 ...      G600  3600.0       MAKE_BELIEVE"
+      ]
+     },
+     "execution_count": 57,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "lfield.observing[0:5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load galaxies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "sdss_file = path+'/'+field[0]+'/'+field[0]+'_SDSS.fits'\n",
+    "sdss_tab = Table.read(sdss_file)\n",
+    "# VSTACK\n",
+    "#hectospec_file = xcasbahu.get_filename(field,'HECTOSPEC')\n",
+    "lfield.galaxies = sdss_tab"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 59,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "&lt;Table length=5&gt;\n",
+       "<table id=\"table4490986832\">\n",
+       "<thead><tr><th>RA</th><th>DEC</th><th>Z</th><th>Z_ERR</th><th>SDSS_MAG [5]</th><th>SDSS_MAGERR [5]</th><th>TELESCOPE</th><th>INSTRUMENT</th><th>FLUX [5000]</th><th>SIG [5000]</th><th>WAVE [5000]</th></tr></thead>\n",
+       "<thead><tr><th>float64</th><th>float64</th><th>float64</th><th>float64</th><th>float64</th><th>float64</th><th>str80</th><th>str80</th><th>float32</th><th>float32</th><th>float64</th></tr></thead>\n",
+       "<tr><td>212.363198097</td><td>26.3766678271</td><td>1.88746607304</td><td>0.000153284345288</td><td>21.94763 .. 0.0</td><td>1.772229 .. 0.0</td><td>SDSS 2.5-M</td><td>BOSS</td><td>13.2035 .. 0.0</td><td>3.4688 .. 0.0</td><td>3592.52587891 .. 0.0</td></tr>\n",
+       "<tr><td>212.430686007</td><td>26.1133116853</td><td>0.673991203308</td><td>0.000221190101001</td><td>24.90705 .. 0.0</td><td>4.125149 .. 0.0</td><td>SDSS 2.5-M</td><td>BOSS</td><td>-0.0600585 .. 0.0</td><td>2.7654 .. 0.0</td><td>3592.52587891 .. 0.0</td></tr>\n",
+       "<tr><td>212.343915164</td><td>26.4916239719</td><td>0.321283131838</td><td>4.98580266139e-05</td><td>18.42358 .. 0.0</td><td>0.0526542 .. 0.0</td><td>SDSS 2.5-M</td><td>BOSS</td><td>-1.64116 .. 0.0</td><td>3.06158 .. 0.0</td><td>3590.87353516 .. 0.0</td></tr>\n",
+       "<tr><td>212.385747743</td><td>26.491183107</td><td>0.67558658123</td><td>0.000228605858865</td><td>19.84492 .. 0.0</td><td>0.1783652 .. 0.0</td><td>SDSS 2.5-M</td><td>BOSS</td><td>2.83022 .. 0.0</td><td>3.29004 .. 0.0</td><td>3592.52587891 .. 0.0</td></tr>\n",
+       "<tr><td>212.48142489</td><td>26.4662754507</td><td>0.130176991224</td><td>1.88579542737e-05</td><td>21.54407 .. 0.0</td><td>0.5453481 .. 0.0</td><td>SDSS 2.5-M</td><td>BOSS</td><td>3.19202 .. 0.0</td><td>2.76537 .. 0.0</td><td>3592.52587891 .. 0.0</td></tr>\n",
+       "</table>"
+      ],
+      "text/plain": [
+       "<Table length=5>\n",
+       "      RA           DEC      ...   SIG [5000]       WAVE [5000]     \n",
```

Comment:
I see; Then I'm fine with this first pass. Will let you know if I find issues after using it a bit.